mne/viz/tests/test_3d.py

```diff
@@ -141,7 +156,7 @@ def test_plot_evoked_field():
 @testing.requires_testing_data
 @requires_mayavi
 @traits_test
-def test_plot_alignment():
+def test_plot_alignment(tmpdir):
     """Test plotting of -trans.fif files and MEG sensor layouts."""
     # generate fiducials file for testing
     tempdir = _TempDir()
```

Comment:
do not overwrite tempdir

Reply:
I changed it to be `tempdir = str(tmpdir)`. This works around a `pytest` bug where `tmpdir` on older `pytest` is not seen properly as a string by some functions, so casting it explicitly to a string makes some things work better